Mathlib/Topology/Maps/Basic.lean

```diff
@@ -353,16 +363,28 @@ theorem of_isEmpty [h : IsEmpty X] (f : X → Y) : IsOpenMap f := of_nhds_le h.e
 
 end IsOpenMap
 
+lemma isOpenMap_iff_kernImage :
+    IsOpenMap f ↔ ∀ {u : Set X}, IsClosed u → IsClosed (kernImage f u) := by
```

Comment:
Seems like this should get a nice docstring because the `IsClosedMap` version has one (even if you don't have quite as tidy an explanation).